_posts/2019-09-10-plipnick-Living_the_Dream_with_Continuous_Improvements.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+author: plipnick 
+title: Living the Dream with Continuous Improvements
+---
+
+
+<img src="https://technologyandoptimizingprocesses.files.wordpress.com/2019/08/morocco-20-copy-1.jpg"
+width="35%"
+alt="https://technologyandoptimizingprocesses.com/">
+
+“You don’t have to be a genius or a visionary or even a college graduate to be successful. You just need a framework and a dream.”
+– Michael Dell
+
+The Society for Human Resource Management’s 2017 Employee Job Satisfaction Survey stated that only 41% of respondents were extremely satisfied with the work itself. Unfortunately, this is not living the dream. If people are going into the office everyday feeling uninspired about their work, then this is an opportunity for innovation. The great news is that we can create a better work environment for everyone. It starts with creating a continuous improvement culture, and this empowers teams to optimize their processes. What does optimizing processes include? It includes many things, but some of the activities are streamlining, eliminating, and automating processes. If we reduce the time that our teams spend on manual work, then they have time to work on innovative ideas that can often drive revenue. Innovation increases the company’s bottom line, so the organization wins. Also, this work is typically more exciting, so our colleagues win. It creates happier teams, and the company creates a solid competitive advantage due to the increased long term business value.
+
+Creating a continuous improvement culture starts with a mind shift change, so everyone starts thinking about the ways they want to improve processes. Also, it helps to have a strategy, so the teams know how to get started with their first continuous improvement project. In addition, it is a good idea to have someone coaching the team that has had previous experience in the continuous improvement space.
+
+There are 3 high level components to a continuous improvement project:
+
+A goal
```

Comment:
If you want to bullet point these, prefix with a "- ". Two lines that are not marked as lists with no space in-between will render without a CR/LF in markdown.
You can see examples in the link I referenced above.

Reply:
Good call :-)